app/models.py

```diff
@@ -1267,6 +1270,7 @@ def get_link(self):
 EMAIL_PROVIDERS = [SES_PROVIDER]
 PROVIDERS = SMS_PROVIDERS + EMAIL_PROVIDERS
 
+NotificationType = Literal["email", "sms", "letter"]
 NOTIFICATION_TYPE = [EMAIL_TYPE, SMS_TYPE, LETTER_TYPE]
```

Comment:
This seems duplicating too - maybe I'm missing something...

Reply:
See my comment here: https://github.com/cds-snc/notification-api/pull/1624/files#r990332042